app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,52 @@
+<div class="section">
+<div class="container">
+<div class="columns is-centered">
+
+<div class="column is-4">
+<h2 class="title is-2"> Sign Up </h2>
+
+<%= simple_form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+<%= f.error_notification %>
+
+<div class="field">
+<div class="control">
+<%= f.input :name, required: true, autofocus: true, input_html: { class: "input" }, wrapper: false, label_html: { class: "label" } %>
+</div>
+</div>
+
+<div class="field">
+<div class="control">
+<%= f.input :username, required: true, input_html: { class: "input" }, wrapper: false, label_html: { class: "label" } %>
+</div>
+</div>
+
+<div class="field">
+<div class="control">
+<%= f.input :email, required: true, input_html: { class: "input" }, wrapper: false, label_html: { class: "label" } %>
+</div>
+</div>
+
+<div class="field">
+<div class="control">
+<%= f.input :password, required: true, input_html: { class: "input" }, wrapper: false, label_html: { class: "label" }, hint: ("#{@minimum_password_length} characters minimum" if @minimum_password_length)  %>
+</div>
```

Comment:
- [ ] at the end of this line there's an If statement, put the if in a helper method that you call here and render this line within the helper method instead of having the if statement directly in the view